enc.go

```diff
@@ -182,7 +182,7 @@ func (c *EncodedConn) QueueSubscribe(subject, queue string, cb Handler) (*Subscr
 }
 
 // Internal implementation that all public functions will use.
-func (c *EncodedConn) subscribe(subject, queue string, cb Handler) (*Subscription, error) {
+func (c *EncodedConn) subscribe(subject, queue string, cb Handler, opts ...SubscribeOption) (*Subscription, error) {
```

Comment:
For now I am ok not upgraded EncodedConn. We need a better direction for it anyway.

Reply:
I agree... most clients don't support encoded conns and didn't see it used much.  NKIT may be better suited.